webpack/assets/javascripts/react_app/common/sessionStorage.js

```diff
@@ -0,0 +1,23 @@
+if (!window.sessionStorage) {
```

Comment:
It's better if you're looking to **add** properties to existing objects. in this case it is not needed. 
this line is used to mock sessionStorage in tests. i'm trying to find more elegant solutions to this.